### DISCUSSION TITLE
Mail with images

### DISCUSSION THREAD

**Discussion by adiogo42** [gophish/gophish#3286] [Q&A]
how can i create a mail template with images? when i try to create the mail template with images  *.jpeg , the images don't appear in mail create or campaign.

**Comment by zaphod13** [comment]
This is an older question, but here are the steps to have images in your email templates:

1. From the Edit Template form, click the Add Files button, and upload an image to the template.
2. In the HTML tab, make sure that you are viewing the rendered HTML (not source code), place cursor where you want the image, and click the image button in the text-editing toolbar.
3. In the URL field, enter in "cid:" and then the name of the image file that you uploaded to the template (i.e. cid:myimage.gif)
4. set any other image parameters, and then click OK to save the image settings.
5. In the preview of the email body it will show a broken image icon, but when the email is sent the uploaded image should display correctly.
6. click the Save Template button to save the email template.
